---
title: Precise phylogenetic analysis of microbial isolates and genomes from metagenomes
  using PhyloPhlAn 3.0
authors:
- Francesco Asnicar
- Andrew Maltez Thomas
- Francesco Beghini
- Claudia Mengoni
- Serena Manara
- Paolo Manghi
- Qiyun Zhu
- Mattia Bolzan
- Fabio Cumbo
- Uyen May
- Jon G Sanders
- Moreno Zolfo
- Evguenia Kopylova
- Edoardo Pasolli
- Rob Knight
- Siavash Mirarab
- Curtis Huttenhower
- Nicola Segata
date: '2020-05-01'
publishDate: '2025-05-15T19:54:25.006898Z'
publication_types:
- article-journal
publication: '*Nat. Commun.*'
abstract: Microbial genomes are available at an ever-increasing pace, as cultivation
  and sequencing become cheaper and obtaining metagenome-assembled genomes (MAGs)
  becomes more effective. Phylogenetic placement methods to contextualize hundreds
  of thousands of genomes must thus be efficiently scalable and sensitive from closely
  related strains to divergent phyla. We present PhyloPhlAn 3.0, an accurate, rapid,
  and easy-to-use method for large-scale microbial genome characterization and phylogenetic
  analysis at multiple levels of resolution. PhyloPhlAn 3.0 can assign genomes from
  isolate sequencing or MAGs to species-level genome bins built from >230,000 publically
  available sequences. For individual clades of interest, it reconstructs strain-level
  phylogenies from among the closest species using clade-specific maximally informative
  markers. At the other extreme of resolution, it scales to large phylogenies comprising
  >17,000 microbial species. Examples including Staphylococcus aureus isolates, gut
  metagenomes, and meta-analyses demonstrate the ability of PhyloPhlAn 3.0 to support
  genomic and metagenomic analyses.
---
