---
title: Metagenomic analysis of colorectal cancer datasets identifies cross-cohort
  microbial diagnostic signatures and a link with choline degradation
authors:
- Andrew Maltez Thomas
- Paolo Manghi
- Francesco Asnicar
- Edoardo Pasolli
- Federica Armanini
- Moreno Zolfo
- Francesco Beghini
- Serena Manara
- Nicolai Karcher
- Chiara Pozzi
- Sara Gandini
- Davide Serrano
- Sonia Tarallo
- Antonio Francavilla
- Gaetano Gallo
- Mario Trompetto
- Giulio Ferrero
- Sayaka Mizutani
- Hirotsugu Shiroma
- Satoshi Shiba
- Tatsuhiro Shibata
- Shinichi Yachida
- Takuji Yamada
- Jakob Wirbel
- Petra Schrotz-King
- Cornelia M Ulrich
- Hermann Brenner
- Manimozhiyan Arumugam
- Peer Bork
- Georg Zeller
- Francesca Cordero
- Emmanuel Dias-Neto
- Jo√£o Carlos Setubal
- Adrian Tett
- Barbara Pardini
- Maria Rescigno
- Levi Waldron
- Alessio Naccarati
- Nicola Segata
date: '2019-04-01'
publishDate: '2025-05-15T19:54:25.037855Z'
publication_types:
- article-journal
publication: '*Nat. Med.*'
abstract: Several studies have investigated links between the gut microbiome and colorectal
  cancer (CRC), but questions remain about the replicability of biomarkers across
  cohorts and populations. We performed a meta-analysis of five publicly available
  datasets and two new cohorts and validated the findings on two additional cohorts,
  considering in total 969 fecal metagenomes. Unlike microbiome shifts associated
  with gastrointestinal syndromes, the gut microbiome in CRC showed reproducibly higher
  richness than controls (P < 0.01), partially due to expansions of species typically
  derived from the oral cavity. Meta-analysis of the microbiome functional potential
  identified gluconeogenesis and the putrefaction and fermentation pathways as being
  associated with CRC, whereas the stachyose and starch degradation pathways were
  associated with controls. Predictive microbiome signatures for CRC trained on multiple
  datasets showed consistently high accuracy in datasets not considered for model
  training and independent validation cohorts (average area under the curve, 0.84).
  Pooled analysis of raw metagenomes showed that the choline trimethylamine-lyase
  gene was overabundant in CRC (P = 0.001), identifying a relationship between microbiome
  choline metabolism and CRC. The combined analysis of heterogeneous CRC cohorts thus
  identified reproducible microbiome biomarkers and accurate disease-predictive models
  that can form the basis for clinical prognostic tests and hypothesis-driven mechanistic
  studies.
---
